jetbrains-core/src/software/aws/toolkits/jetbrains/utils/execution/steps/StepExecutor.kt

```diff
@@ -109,6 +109,10 @@ class StepExecutor(
         override fun createChildEmitter(stepName: String, hidden: Boolean): StepEmitter = StepEmitterWrapper(delegate.createChildEmitter(stepName, hidden))
 
         override fun emitMessage(message: String, isError: Boolean) {
+            if (ApplicationManager.getApplication().isUnitTestMode) {
+                print(message)
```

Comment:
Why do we explicitly print this here?

Reply:
tired of not seeing any output on test failure and too much work to figure out why LOG.info isn't working at the moment